src/components/CospendNavigation.vue

```diff
@@ -194,6 +233,10 @@ export default {
 	beforeMount() {
 	},
 	methods: {
+		toggleArchivedProjects(showArchivedProjects = false) {
+			this.showArchivedProjects = showArchivedProjects
+			this.deselectProject()
+		},
```

Comment:
```suggestion
		toggleArchivedProjects() {
			this.showArchivedProjects = !this.showArchivedProjects
			emit('deselect-project')
		},
```
Subscribe to this event in `App` to call deselectProject there.